contrib/platform/src/com/sun/jna/platform/win32/Psapi.java

```diff
@@ -320,4 +338,79 @@ class PERFORMANCE_INFORMATION extends Structure {
         public DWORD ProcessCount;
         public DWORD ThreadCount;
     }
+
+    @FieldOrder({"Flags"})
+    class PSAPI_WORKING_SET_EX_BLOCK extends Structure {
+        public ULONG_PTR[2] Flags = new ULONG_PTR[Native.POINTER_SIZE == 8 ? 1 : 2];
+        private Long innerValue;
+        private void setInnerValue() {
```

Comment:
A better/simpler way to handle this is to override `read()` and just always write the value, based on your logic below, without worrying about null checks.